src/classic/types/ReactPropTypes.js

```diff
@@ -86,19 +86,20 @@ var ReactPropTypes = {
 };
 
 function createChainableTypeChecker(validate) {
-  function checkType(isRequired, props, propName, componentName, location) {
+  function checkType(isRequired, props, propName, componentName, location, propFullName) {
```

Comment:
And you run lint and fix all the cases in this file - you have a bunch of lines that are now going to be > 80 (it's just a warning but we'll get to a point where we're more strict and I'd rather try to catch them as they come in.

Since we just care about this file `./node_modules/.bin/eslint  src/classic/types/ReactPropTypes.js` should work.